server/graphql/v2/object/AccountCollectionStats.ts

```diff
@@ -0,0 +1,41 @@
+import { GraphQLNonNull, GraphQLObjectType } from 'graphql';
+
+import { TransactionKind } from '../../../constants/transaction-kind';
+import { TransactionTypes } from '../../../constants/transactions';
+import queries from '../../../lib/queries';
+import { getTimeSeriesFields } from '../interface/TimeSeries';
+import { resultsToAmountNode } from '../object/HostMetricsTimeSeries';
+import { TimeSeriesAmount } from '../object/TimeSeriesAmount';
+import { TimeSeriesCount } from '../object/TimeSeriesCount';
+
+export const AccountCollectionStats = new GraphQLObjectType({
+  name: 'AccountCollectionStats',
+  description: 'Account collection stats',
+  fields: () => ({
+    ...getTimeSeriesFields(),
+    contributionsCountTimeSeries: {
+      type: new GraphQLNonNull(TimeSeriesCount),
+      description: 'Time series of the number of contributions to these accounts',
+      resolve: async ({ collectiveIds, dateFrom, dateTo, timeUnit }) => {
+        const kind = [TransactionKind.CONTRIBUTION];
+        const transactionParams = { type: TransactionTypes.CREDIT, kind, dateFrom, dateTo, collectiveIds };
+        const countNodes = collectiveIds.length
+          ? await queries.getTransactionsCountTimeSeries(timeUnit, transactionParams)
+          : [];
+        return { dateFrom, dateTo, timeUnit, nodes: countNodes };
+      },
+    },
+    totalReceivedTimeSeries: {
```

Comment:
Same here `totalTransactionsAmountTimeSeries`

Reply:
renamed it to `transactionsAmountTimeSeries`, I think it fits better with `transactionsCountTimeSeries`?